src/templates/robots.ts

```diff
@@ -0,0 +1,29 @@
+import type {
+  TemplateProps,
+  TemplateRenderProps,
+  GetPath,
+  TemplateConfig,
+} from "@yext/pages";
+
+export const config: TemplateConfig = {
+  // The name of the feature. If not set the name of this file will be used (without extension).
+  // Use this when you need to override the feature name.
+  name: "robots",
+};
+
+/**
+ * Defines the path that the generated file will live at for production.
+ *
+ * NOTE: This currently has no impact on the local dev path. Local dev urls currently
+ * take on the form: featureName/entityId
+ */
+export const getPath: GetPath<TemplateProps> = () => {
+  return `robots.txt`;
```

Comment:
What is this file for ?

Reply:
This is for the robots.txt page [here](https://directoryqa-seemly--realistic--impala-pgsdemo-com.preview.pagescdn.com/robots.txt), which has rules set out for web crawlers. This was included in project phase 1 for SEO purposes.